reset_security_server.sh

```diff
@@ -59,19 +57,17 @@ run_ansible_script() {
     fi
 }
 
-create_api_keys() {
+add_public_key() {
     names=$(echo "$1" | tr "," "\n")
     for name in $names
     do
-        printf "\nCreating api-key for LXD container %s\n" "$name"
-        FILE=api-key-$name.txt
-        lxc exec "$name" -- curl -X POST -u xrd:secret --retry 30 --retry-connrefused --silent "$2" --data "$3" --header "$4" -k -o "$FILE"
-        lxc exec "$name" -- cat api-key-"$name".txt | jq '.key'
+        printf "\nAdding public key to LXD container %s\n" "$name"
+        lxc exec "$name" -- bash -c "echo \"$(cat "$2")\" > .ssh/authorized_keys"
```

Comment:
Here it failed for me, with message:

```Adding public key to LXD container ss3
bash: .ssh/authorized_keys: No such file or directory```

Probably because the ``.ssh`` folder is not present in container yet.

Command line was:
```./reset_security_server.sh  -n ss3 -h ../../X-Road/ansible/hosts/lxd_hosts.txt  -a ../../X-Road/ansible -k ~/.ssh/id_ed25519.pub```

and it would seem to me that the content of the public key should be taken from the host and then passed through to  a container.

Reply:
Works for me with the following command:

./reset_security_server.sh -n ss3,ss4 -h ../X-Road/ansible/hosts/lxd_hosts.txt -a ../X-Road/ansible -k ../../work/gofore_public_key

The process:

1. Deletes the given lxd containers if present
2. Installs security servers to the containers using Ansible script and hosts file
3. Copies your public key to ".ssh/authorized_keys" on the containers

Reply:
Probably adding "mkdir .ssh" would solve the problem, I will try adding that there.

Reply:
Added in the code the part that creates the folder when it does not exist